### DISCUSSION TITLE
Main "competitor" - Tracktak

### DISCUSSION THREAD

**Discussion by KhanPhelan** [agrlh/fininsights#8] [Ideas]
Very good site, uses the data from here:
https://eodhistoricaldata.com/pricing
Heavy focus on Discounted Cashflow Model, and will be switch to paid-version once Beta is over
https://tracktak.com/

**Comment by ZuitAMB** [comment]
https://aktienfinder.net 
![Bild_2021-07-08_025735](https://user-images.githubusercontent.com/56884080/124846035-40326680-df98-11eb-816b-3ecd63e8ad1f.png)
Filtering/Sorting/Visualizing/Prediction of stock fundamental data etc. and the guy pushes his site via [youtube](https://www.youtube.com/channel/UCp4NH9dmMSN2l8PPeVHfEsQ).
